hw06/task1.js

```diff
@@ -0,0 +1,32 @@
+//Function declaration
+
+function calculateRectangleArea (width1, height1) {
+    //console.log(sumOfWidthHeight)
+    const rectangleArea = width1 * height1
+    return rectangleArea
+}
+const widthHeight = calculateRectangleArea (100,450)
```

Comment:
I made a comment about this variable name on previous review. Pick better name like `rectagleArea`

Reply:
https://github.com/narinegi/aqa-advanced/pull/8/commits/d17e73752919facecd364f40970b5ce433ca0b12